amundsen_application/api/metadata/v0.py

```diff
@@ -674,7 +674,7 @@ def _get_related_dashboards_metadata(*, url: str) -> Dict[str, Any]:
     results_dict['status_code'] = status_code
 
     if status_code != HTTPStatus.OK:
-        message = 'Encountered error: Related Dashboard Metadata request failed'
+        message = f'Encountered {status_code} Error: Related dashboard metadata request failed'
```

Comment:
This message will show in the UI if there is a failure from the metadata service.
**Example:**
![image](https://user-images.githubusercontent.com/1790900/85486482-8c725a80-b57f-11ea-9189-c95db49d8db1.png)

Reply:
curious: do you use the open source version to mimic the issue or just override it with FE API endpoint?

Reply:
I updated the code in Flask to mimic returning an error instead of the correct response -- just like I would use it to return dummy data as well if there's some specific case we have to test but don't have the real data in our environment for.